clang/lib/Sema/SemaDeclAttr.cpp

```diff
@@ -1481,6 +1481,17 @@ static void handleOwnershipAttr(Sema &S, Decl *D, const ParsedAttr &AL) {
     break;
   }
 
+  // Allow only pointers to be return type for functions with ownership_takes
+  // attribute. This matches with current OwnershipAttr::Takes semantics
+  if (K == OwnershipAttr::Returns) {
```

Comment:
This comment seems to be talking about `ownership_takes`, and not `ownership_returns`?